mainwindow.h

```diff
@@ -26,16 +25,15 @@ private slots:
 
 private:
     Ui::MainWindow *ui;
-    Choices player1Choice;
-    Choices player2Choice;
     int roundCount = 1;
     int winCount = 0;
     int tieCount = 0;
     int loseCount = 0;
-    void display_choice();
-    Choices generate_choice();
     QString get_result(Choices p1, Choices p2);
     void display_round();
     void display_result();
+
+    Player* player1 = new Player;
```

Comment:
it's okay for now, but if we support multiple players then I think we should change this

Reply:
ok
but let's just keep it that way for now.

Reply:
Gonna mark this one as done.